dev/com.ibm.ws.transport.http2_fat/test-applications/H2FATDriver.war/src/http2/test/driver/war/servlets/H2FATDriverServlet.java

```diff
@@ -2125,8 +2249,15 @@ public void testContinuationFrameOnClosedStream(HttpServletRequest request,
         CountDownLatch blockUntilConnectionIsDone = new CountDownLatch(1);
         Http2Client h2Client = getDefaultH2Client(request, response, blockUntilConnectionIsDone);
 
-        byte[] debugData = "CONTINUATION frame received on a closed stream".getBytes();
-        FrameGoAway errorFrame = new FrameGoAway(0, debugData, STREAM_CLOSED, 1, false);
+        // TODO Need to look more into this, not exactly sure why it's sending a go away frame since the stream should be closed
```

Comment:
Need to check if this todo is still unresolved

Reply:
This requires some deep investigation as I'm not familiar with the Frames. Will take some time.

Reply:
I'll create a todo issue to investigate post beta

Reply:
Linked to post beta issue, we can create sub issues once we start working on it: https://github.com/OpenLiberty/open-liberty/issues/32386